libbeat/processors/add_docker_metadata/add_docker_metadata.go

```diff
@@ -113,7 +113,7 @@ func buildDockerMetadataProcessor(log *logp.Logger, cfg *conf.C, watcherConstruc
 	}
 
 	reader, err := initCgroupPaths(resolve.NewTestResolver(config.HostFS), false)
-	if err != nil {
+	if err != nil && !errors.Is(err, cgroup.ErrCgroupsMissing) {
```

Comment:
nit - since an error results in a nil `reader`, which requires us to perform `d.cgreader == nil` checks, it could be best to create a `NoOpCGReader` which returns an empty `cgroup.PathList` when this error occur.

Should be straightforward given we already have `CGReader` interface [1]

[1] - https://github.com/fearful-symmetry/beats/blob/03fed63f532875935a1ca13e5c7a43ca59315694/libbeat/processors/cgroups.go

Reply:
Yeah, will do this in a follow-up PR, since it's a tad cleaner

Reply:
thanks :)